### ISSUE TITLE
top(ic) overlays scroll bars

### ISSUE THREAD

**Issue by SimJoSt** [glowing-bear/glowing-bear#873]
The top bar with the glowing-bear logo, the channel topic as well as the settings and log out button, overlays both the top end of the scroll bars: The one for the list of buffers, the current buffer and the one for the nick list. Of course provided the mentioned element/list is long enough to have a scroll bar displayed.

It's mostly a surficial inconvenience, but with a very long list, it is possible to completely lose the scroll bar under the top bar, which makes grabbing it with the mouse impossible.

**Comment by lorenzhs** [comment]
Yeah, this is sort of a known issue, our layouting isn't really all that well done. You're right that it's annoying. My utter CSS non-skill doesn't really help ;) We should fix this

**Comment by Vaesper** [comment]
I recently found the reason for this as it annoyed me as well. Top bar is positioned as static, the other divs are height: 100%. With the [CSS calc() function](https://css-tricks.com/a-couple-of-use-cases-for-calc/) it can be fixed, but I don't have time to work on a PR right now.
